spec/factories/keywords.rb

```diff
@@ -1,9 +1,11 @@
 # frozen_string_literal: true
 
+require 'faker'
+
 FactoryBot.define do
   factory :keyword do
     work_version { nil }
-    label { 'MyKeyword' }
+    label { "#{Faker::Food.spice} #{Faker::Food.ingredient}" }
     uri { 'http://example.org/uri' }
     cocina_type { 'place' }
   end
```

Comment:
other tests that create multiple keywords using factories need to be unique now, or else they will fail with the new unique constraint we added